components/PointsBox.js

```diff
@@ -9,23 +9,25 @@ const PointsBox = ({ props }) => {
     <View>
       {semester === "Fall Semester" ? (
         <View style={styles.pointsBox}>
-          <Text style={styles.bold}>FALL POINTS</Text>
+          <Text style={styles.title}>FALL POINTS</Text>
```

Comment:
The title thing is not centering it. Why aren't all the titles the same? Just leave it as styles.center